lib/solargraph.rb

```diff
@@ -47,7 +47,7 @@ class InvalidRubocopVersionError < RuntimeError;  end
   autoload :Parser,           'solargraph/parser'
   autoload :RbsMap,           'solargraph/rbs_map'
   autoload :GemPins,          'solargraph/gem_pins'
-  autoload :Cache,            'solargraph/cache'
+  autoload :PinCache,         'solargraph/pin_cache'
```

Comment:
Not a fantastic name ("GemCache?"), but it helped me keep this distinct in my head from ApiMap::Cache